docker/policystream

```diff
@@ -36,10 +37,27 @@ RUN . /usr/local/bin/activate && \
 
 RUN mkdir /output
 
+# Update git since the base version does not allow for --initial-branch
+RUN apt-get install wget autoconf libz-dev gettext -y && \
+    wget https://www.kernel.org/pub/software/scm/git/git-2.37.2.tar.gz && \
+    tar -zxf git-2.37.2.tar.gz && \
+    cd git-2.37.2 && \
+    make configure && \
+    ./configure --prefix=/usr && \
+    make all && \
+    make install && \
+    git --version
+
+
 # Install c7n-policystream
 ADD tools/c7n_policystream /src/tools/c7n_policystream
 RUN . /usr/local/bin/activate && cd tools/c7n_policystream && $HOME/.poetry/bin/poetry install
 
+# Verify the install
+#  - policystream is not in ci due to libgit2 compilation needed
+#  - as a sanity check to distributing known good assets / we test here
+RUN . /usr/local/bin/activate && pytest -p "no:terraform" tools/c7n_policystream
```

Comment:
why is this is being reintroduced?

Reply:
I saw that it was in the distroless dockerfile on master: https://github.com/cloud-custodian/cloud-custodian/blob/master/docker/policystream-distroless#L43-L46 but i can remove it

Reply:
yeah, we should yank it from both, this stuff pre dated pygit2 distributing binary wheels across platforms, but it does now.